packages/pagebuilder/lib/ContentTypes/ButtonItem/__tests__/buttonItem.spec.js

```diff
@@ -112,7 +112,7 @@ test('clicking button with internal link goes to correct destination', () => {
 
 test('clicking button without link', () => {
     const buttonItemProps = {
-        link: ' ',
+        link: undefined,
```

Comment:
we should probably get a default event for link.

Reply:
what do you mean?

Reply:
instead of undefined link maybe use "javascript:undefined"